packages/venia-concept/src/index.js

```diff
@@ -36,12 +38,35 @@ const authLink = setContext((_, { headers }) => {
     };
 });
 
+// https://www.apollographql.com/docs/link/links/error/
+const errorLink = onError(({ graphQLErrors, networkError }) => {
+    if (graphQLErrors)
+        graphQLErrors.forEach(({ message, locations, path }) =>
+            console.log(
+                `[GraphQL error]: Message: ${message}, Location: ${locations}, Path: ${path}`
+            )
+        );
+    if (networkError) console.log(`[Network error]: ${networkError}`);
+});
+
 // @see https://www.apollographql.com/docs/link/composition/.
 const apolloLink = ApolloLink.from([
     new MutationQueueLink(),
-    // by default, RetryLink will retry an operation five (5) times.
-    new RetryLink(),
+    new RetryLink({
+        delay: {
+            initial: 300,
+            max: Infinity,
+            jitter: true
+        },
+        attempts: {
+            max: 5,
+            retryIf: () => {
+                return navigator.onLine;
+            }
```

Comment:
Does this actually work? I would have expected something on top of the default:

```suggestion
            retryIf: (error) => !!error && navigator.onLine 
```